pkg/agent/util/net_unix.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build linux darwin
+
+package util
+
+import (
+	"fmt"
+	"net"
+
+	"golang.org/x/sys/unix"
+	"k8s.io/klog"
+
+	"github.com/vishvananda/netlink"
+)
+
+// GetLocalNodeAddr return a local IP/mask that is on the path of default route.
+func GetDefaultLocalNodeAddr(localIP net.IP) (*net.IPNet, netlink.Link, error) {
```

Comment:
function name and implementation don't match its comment.

Reply:
done